### ISSUE TITLE
2.5.8 Rozmiar celu

### ISSUE THREAD

**Issue by zwiastunsw** [irdpl/wcag#19]
## Oryginał
````html
    <h4>Target Size (Minimum)</h4>
    <p class="conformance-level">AA</p>
    <p class="change">New</p>
    <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
    <ul>
      <li><strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> 
      of at least 24 CSS pixels to every adjacent target;</li>
      <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page 
      that meets this criterion.</li>
      <li><strong>Inline:</strong> The target is in a sentence, or is in a bulleted or numbered list, 
      or its size is otherwise constrained by the line-height of non-target text;</li>
      <li><strong>User agent control:</strong> The size of the target is determined by the user agent 
      and is not modified by the author;</li>
      <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> 
      or is legally required for the information being conveyed;</li>
    </ul>
    <p class="note">Targets that allow for values to be selected spatially based on position within 
    the target are considered one target for the purpose of the success criterion. Examples include sliders 
    with granular values, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p>
    <p class="note">For inline targets the line-height should be interpreted as perpendicular to the flow of text. 
    For example, in a language displayed top to bottom, the line-height would be horizontal.</p>
````
## Wstępna propozycja
````html
    <h4>Rozmiar celu (minimalny)</h4>
    <p class="conformance-level">AA</p>
    <p class="change">Nowe</p>
    
    <p>Rozmiar <a>celu</a> <a>punktu dotykowego</a> wynosi co najmniej 24 na 24 <a>piksele CSS</a>, chyba że: </p>
    <ul>
      <li><strong>Odstępy:</strong> Cele nie nachodzą na siebie i są oddzielone odstępami o wielkości 
      co najmniej 24 <a>pikseli CSS</a>;</li>
      <li><strong>Odpowiednik:</strong> Efekt może być osiągnięty na tej samej stronie za pomocą innej kontrolki, 
      która spełnia to kryterium.</li>
      <li><strong>Śródliniowe:</strong> Cel znajduje się w zdaniu, na liście wypunktowanej bądź numerowanej, 
      lub jego rozmiar jest w inny sposób ograniczony przez wysokość linii tekstu niebędącego celem;</li>
      <li><strong>Kontrolowany przez program użytkownika:</strong> Rozmiar celu określony jest przez 
      <a>program użytkownika</a> i nie jest modyfikowany przez autora;</li>
      <li><strong>Istotny:</strong> Sposób przedstawienia celu jest <a>niezbędny</a> 
      albo jest prawnie wymagany przy przekazywaniu informacji.</li>
    </ul>
    <p class="note">Cele, które pozwalają na to by wartości były wybierane przestrzennie w oparciu o pozycję wewnątrz 
    celu, są na potrzeby tego kryterium sukcesu uznawane są za jeden cel. Przykładami mogą być suwaki z drobnymi 
    wartościami, próbniki kolorów prezentujące gradient koloru lub pola edycji, w których umieszczany jest kursor.</p>
    <p class="note">Dla celów śródliniowych wysokość linii należy rozumieć jako prostopadłą w stosunku do układu tekstu. 
    Na przykład, w językach zapisywanych od góry do dołu wysokość linii należy rozumieć jako poziomą 
    w stosunku do układu tekstu.</p>
````
## Rzeczy do rozważenia
- **terminy słownikowe**. Rozważ spójność użytych terminów słownikowych z definicją w słowniku: _celu_, _punktu dotykowego_, _piksele CSS_, _pikseli CSS_, _niezbędny_
- **spójność z WCAG 2.1**. Rozważ spójność z KS 2.5.5 Rozmiar celu dotykowego 
---

**Comment by BartekStepien** [comment]
Cel punktu dotykowego ma wymiary minimum 24x24 piksele, poza poniższymi wyjątkami:
Odstępy: cel nie nachodzi na żaden inny cel, a jego offset wynosi minimum 24 piksele CSS;

**Comment by speplinski** [comment]
<strong>Odstępy:</strong> Cele nie nachodzą na siebie i są oddzielone odstępami o wielkości co najmniej 24 <a>pikseli CSS</a>;

W oryginale jest liczba pojedyncza (target)
<strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;

Może tak:
<strong>Odstęp:</strong> Cel nie nachodzi na żaden inny cel i ma <a>odstęp</a> co najmniej 24 pikseli CSS do każdego sąsiedniego celu;

**Comment by speplinski** [comment]
<strong>Śródliniowe:</strong> Cel znajduje się w zdaniu, na liście wypunktowanej bądź numerowanej, lub jego rozmiar jest w inny sposób ograniczony przez wysokość linii tekstu niebędącego celem;

`Śródliniowe` odnosi się bezpośrednio do linii tekstu, ew. można użyć `w linii`, jako ogólniej stosowane, odnoszące się do układu elementów na płaszczyźnie. 

Może tak:
<strong>Śródliniowe:</strong> Cel znajduje się w zdaniu, na liście z wypunktowaniem lub numeracją, lub jego rozmiar jest ograniczony przez wysokość wiersza tekstu nienależącego do celu;

Nie wiem czy nie lepiej byłoby zamiast `Cel znajduje się w zdaniu` użyć `Cel znajduje się w tekście`...

**Comment by BartekStepien** [comment]
Ja bym użył słowa "offset" bo to jest termin, którego wyjaśnienie jest w wcag. Wówczas mamy dość precyzyjny zapis kryterium

**Comment by m-kowalski** [comment]
> [!IMPORTANT]
> Zmiana treści kryterium sukcesu.
> https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#258-target-size-minimum-aa
> https://github.com/w3c/wcag/blob/main/guidelines/sc/22/target-size-minimum.html

## HTML

```html
<h4>Target Size (Minimum)</h4>
<p class="conformance-level">AA</p>
<p class="change">New</p>
<p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
<ul>
    <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a>bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target;</li>
    <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion;</li>
    <li><strong>Inline:</strong> The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;</li>
    <li><strong>User agent control:</strong> The size of the target is determined by the <a>user agent</a> and is not modified by the author;</li>
    <li><strong>Essential:</strong> A particular <a>presentation</a> of the target is <a>essential</a> or is legally required for the information being conveyed.</li>
</ul>
<p class="note">Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p>
<p class="note">For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed vertically, the line-height would be horizontal.</p>
```

## Tekst

**Target Size (Minimum)**

The size of the _target_ for _pointer inputs_ is at least 24 by 24 _CSS pixels_, except where:
- **Spacing:** Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the _bounding box_ of each, the circles do not intersect another target or the circle for another undersized target;
- **Equivalent:** The function can be achieved through a different control on the same page that meets this criterion;
- **Inline:** The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;
- **User agent control:** The size of the target is determined by the _user agent_ and is not modified by the author;
- **Essential:** A particular _presentation_ of the target is _essential_ or is legally required for the information being conveyed.

> Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders with granular values, color pickers displaying a gradient of colors, or editable areas where you position the cursor.

> For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed top to bottom, the line-height would be horizontal.
